scripts/caretaking/cherrypick-patch.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+/**
+ * This script displays the commands that should be run to cherry-pick merged changes
+ * into the current patch branch.
+ *
+ * Run this command like this: `node scripts/caretaking/cherry-pick-patch.js 2019-07-29` where the
+ * date input is the oldest merged pull request that should be patched.
+ *
+ * An authorization token may optionally be provided to increase the API rate limit. A personal
+ * access token string should be added to the command, e.g.:
+ * node scripts/caretaking/cherrypick-patch.js 2019-07-15 4c32652da9e57b2b2IMATOKENLOL8983B3BK
+ */
+
+/** Required argument: Date of the oldest PR to load, should be in the form of YYYY-MM-DD. */
+const date = process.argv[2];
+if (!date) {
+  throw Error('Missing filter date. Provide a date as YYYY-MM-DD, e.g. 2019-06-30');
+}
+
+/** Optional argument: Github personal access token for increased API rate limit. **/
+const authToken = process.argv[3];
+
+const Octokit = require('@octokit/rest');
+const octokit = new Octokit({auth: authToken});
+
+/** Load the pull requests and output their details, including the commands to cherry-pick. */
+(async function main() {
+  const pullRequests = await getPullRequests('angular', 'components');
+  outputPullRequestData(pullRequests);
+  outputCherryPickCommands(pullRequests);
+})();
+
+/** Loads pull requests that need to be cherry-picked. */
+async function getPullRequests(owner, repo) {
+
+  const excludedLabels = ['target: minor', 'target: major', 'target: master only'];
+  const excludeLabelsQueryStr = excludedLabels.map(l => `-label:"${l}"`).join(' ');
+
+  const query =
+      `repo:${owner}/${repo} base:master is:pr is:merged merged:>${date} ${excludeLabelsQueryStr}`;
+
+  const response = await octokit.search.issuesAndPullRequests({per_page: 100, q: query});
```

Comment:
Even though we rarely might hit that, it looks like this will only work for `100` PRs merged. 

If we ever would hit that it could be difficult to see that some PRs were missed. should we use pagination?